README.md

```diff
@@ -30,7 +37,7 @@ kubectl apply -f https://github.com/knative/serving/releases/latest/download/ser
 - Install net-gateway-api
 
 ```bash
-ko apply -f config/
+ko apply -f third_party/gateway-api/
```

Comment:
Things specific to this repo should probably remain in `config/` 

ie. controller, cluster role, config

I'm indifferent whether the gateway CRDs are in `config/` or `third_party` - ie. in contour we keep things in a separate nested folder under `config`

Probably worth thinking about if we're going to ship the gateway CRDS with the net-gateway-api release

Any yamls we move around we should update the release script

https://github.com/knative-sandbox/net-gateway-api/blob/main/hack/release.sh